plugins/package-managers/go/src/main/kotlin/GoMod.kt

```diff
@@ -484,3 +470,28 @@ internal fun parseWhyOutput(output: String): Set<String> {
 
     return usedModules
 }
+
+/**
+ * Module paths appear as substrings of file system paths in the module cache on the file system.
+ * Go does not rely on the file system to be case-sensitive. For this reason, Go has decided to
+ * replace every uppercase letter in file system paths with an exclamation mark followed by the
+ * letter's lowercase equivalent.
+ *
+ * Details behind the reasoning and implementation in Go can be found in the Go source code at
+ * [module.go](https://github.com/golang/go/blob/master/src/cmd/vendor/golang.org/x/mod/module/module.go)
+ */
```

Comment:
We usually use perma-links, so that these do not  break. I propose to also include some line range as the file is large. Would the following fit?

https://github.com/golang/go/blob/5b6d3dea8744311825fd544a73edb8d26d9c7e98/src/cmd/vendor/golang.org/x/mod/module/module.go#L33-L42C64

Reply:
Done